clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -273,6 +273,7 @@ protected void onJoinComplete(int generation,
      */
     public void poll(long now, long remainingMs) {
         invokeCompletedOffsetCommitCallbacks();
+        now = time.milliseconds();
```

Comment:
Good find. Perhaps to save unneeded calls to system time, we could let `invokeCompletedOffsetCommitCallbacks` return a boolean to indicate whether any callbacks which invoked?

Reply:
@hachikuji Would you please take a look?